cfn-resources/cloud-backup-schedule/docs/apipolicyitemview.md

```diff
@@ -40,7 +40,7 @@ _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormati
 
 #### FrequencyType
 
-Frequency associated with the backup policy item. One of the following values: hourly, daily, weekly or monthly.
+Frequency associated with the backup policy item. One of the following values: hourly, daily, weekly, monthly or yearly.
```

Comment:
knit: keep the comma befor or.  it's called the Oxford comma

 hourly, daily, weekly, monthly, or yearly.

same for all